boilerplate/openshift/golang-osd-operator/golangci.yml

```diff
@@ -21,6 +21,13 @@ linters:
   - gosimple
   - govet
   - ineffassign
+  - misspell
   - staticcheck
   - typecheck
   - unused
+
+linters-settings:
+  misspell:
+    extra-words:
+      - typo: "openshit"
+        correction: "OpenShift"
```

Comment:
Is there a way to mark a diff as your favorite change ever? Asking for a friend.